Makefile

```diff
@@ -11,7 +11,7 @@ all: build build/CMakeLists.txt.copy
 	$(MAKE) --no-print-directory -C build
 
 clean:
-	rm -rf bin build lib msg_gen src/ut_multirobot_sim
+	rm -rf bin build lib msg_gen src/ut_multirobot_sim src/state_switcher_rviz_plugin/lib
```

Comment:
Do we want to include state_switcher_rviz_plugin? I don't think it's relevant to anyone else.

Reply:
I'll remove it from the default build and rviz config, but leave the code for now.